### PR TITLE
Including anakinra in drugs

### PR DESCRIPTION
**Pull request by athril**
### Description of the proposed additions or changes

Adding a paragraph on anakinra - a compound proven to be effective in severe covid-19 treatment.

### Related issues

#602 #604

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
Thank you so much for this submission! Because we are so close to submitting the first version, we are not currently adding new drugs into the text for the first release. However, it sounds like a fascinating route to explore for the second release! I will ping you once we are able to start evaluating new text, but hopefully by early September.

**Comment by RLordan** [comment]
> Thank you so much for this submission! Because we are so close to submitting the first version, we are not currently adding new drugs into the text for the first release. However, it sounds like a fascinating route to explore for the second release! I will ping you once we are able to start evaluating new text, but hopefully by early September.

Very interesting work, but not yet proven as they're are only two cohort studies published to date. Very interesting to see how that drug progresses. It is currently registered in at least 6 clinical trials.

**Comment by athril** [comment]
@rando2 Thank you!
@RLordan I agree, anakinra seems to be a very interesting lead. Please take a look at #601. Among 20 drugs considered, only 
Interferon, lopinavir, ritonavir, and anakinra are highlighted as promising.

**Comment by RLordan** [comment]
> @rando2 Thank you!
> @RLordan I agree, anakinra seems to be a very interesting lead. Please take a look at #601. Among 20 drugs considered, only
> Interferon, lopinavir, ritonavir, and anakinra are highlighted as promising.

Thanks @athril, I certainly will :)

**Comment by agitter** [comment]
When we return to this issue, we should tone down the summary statements.  The concluding sentence of the paper in #601 is
> These results suggest the need of more clinical trials to confirm the efficacy of anakinra to treat patients with severe COVID-19 infection.

That seems to fit the evidence at hand.

Some [trials](https://clinicaltrials.gov/ct2/show/NCT04324021) will also complete in the next month or so.

**Comment by rando2** [comment]
replaced by #757

File: content/20.treatments.md [hunk 1]
```diff
@@ -211,19 +211,18 @@ Brief background on the therapeutic.
...
 
-In two retrospective cohort studies, first at the San Raffaele Hospital in Milan, Italy with 29 patients treated with high dosages of anakinra and 16 controls [@doi:10.1016/S2665-9913(20)30127-2], and second at Groupe Hospitalier Paris Saint-Joseph Paris, France, with 96 participants in France, among which 52 were treated with the drug and 44 were administered standard protocols, anakinra has proven to 
-progressively improve respiratory function, dicrease likelihood of application of invasive mechanical ventilation, and significantly reduce mortality [@doi:10.1016/S2665-9913(20)30164-8].
+In two retrospective cohort studies, first at the San Raffaele Hospital in Milan, Italy with 29 patients treated with high dosages of anakinra and 16 controls [@doi:10.1016/S2665-9913(20)30127-2], and second at Groupe Hospitalier Paris Saint-Joseph Paris, France, with 96 participants in France, among which 52 were treated with the drug and 44 were administered standard protocols, anakinra has proven to progressively improve respiratory function, dicrease likelihood of application of invasive mechanical ventilation, and significantly reduce mortality [@doi:10.1016/S2665-9913(20)30164-8].
```
Comment (athril) [inline content/20.treatments.md:220]: In the recent randomized clinical trial, anakinra was found to be ineffective for COVID-19 and mild-to-moderate pneumonia patients treated intravenously with anakinra, although the studies used a different treatment protocol than the previous studies (half of the dosage, which was admitted intravenously). The authors didn't exclude anakinra effectiveness for severe COVID-19 cases and commented a decreased dosage as the  limitation of their study [@10.1016/S2213-2600(20)30556-7].

**Pull request closed without merging**